database/event_query.py

```diff
@@ -40,6 +41,25 @@ def _query_async(self):
         raise ndb.Return(events)
 
 
+class DistrictTeamsQuery(DatabaseQuery):
```

Comment:
Isn't this already here? 
https://github.com/the-blue-alliance/the-blue-alliance/blob/master/database/team_query.py#L52